debug/pixelorigin.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
```

Comment:
Ah I think this file isn't supposed to be here still?

Reply:
I kinda liked it enough to keep it but I can delete it if you want! This was the debug test file I was referring to. It helped me understand pixel origin and we can try getting it integrated with our UI too